Source/WebCore/html/canvas/GPUCanvasContextCocoa.mm

```diff
@@ -151,18 +151,15 @@ static GPUIntegerCoordinate getCanvasHeight(const GPUCanvasContext::CanvasType&
 {
 }
 
-void GPUCanvasContextCocoa::reshape(int width, int height, int oldWidth, int oldHeight)
+void GPUCanvasContextCocoa::reshape()
 {
-    UNUSED_PARAM(oldWidth);
-    UNUSED_PARAM(oldHeight);
-
     if (auto* texture = m_currentTexture.get()) {
         texture->destroy();
         m_currentTexture = nullptr;
     }
-
-    m_width = static_cast<GPUIntegerCoordinate>(width);
-    m_height = static_cast<GPUIntegerCoordinate>(height);
+    auto newSize = canvasBase().size();
+    m_width = static_cast<GPUIntegerCoordinate>(newSize.width());
+    m_height = static_cast<GPUIntegerCoordinate>(newSize.height());
```

Comment:
This is the only place that changes `m_width` and `m_height` after initializing them in the constructor. Can't we remove these members and replace them by `canvasBase().size()` wherever they are needed?

Reply:
In future when WebGPU  is implemented more rigorous way, these might be used and also constrained.